cmd/juju/machine/show.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package machine
+
+import (
+	"fmt"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"launchpad.net/gnuflag"
+
+	"github.com/juju/juju/cmd/juju/status"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+const showMachineCommandDoc = `
+Show a specified machine on a model:
+
+juju show-machine <machineID[s]>
+
+For example:
+
+juju show-machine 0
+
+or for multiple machines
+(the following will display status for machines 1, 2 & 3):
+
+juju show-machine 1 2 3
+
+Default format is in yaml, other formats can be specified
+with the "--format" option.  Available formats are yaml,
+tabular, and json
+`
+
+// NewShowMachineCommand returns a command that shows details on the specified machine[s].
+func NewShowMachineCommand() cmd.Command {
+	return modelcmd.Wrap(&showMachineCommand{})
+}
+
+// showMachineCommand struct holds details on the specified machine[s].
+type showMachineCommand struct {
+	modelcmd.ModelCommandBase
+	out       cmd.Output
+	isoTime   bool
+	machineId []string
+	api       statusAPI
+}
+
+// Info implements Command.Info.
+func (c *showMachineCommand) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "show-machine",
+		Args:    "<machineID[s]> ...",
```

Comment:
"<machineID> ..."

(see cmd/juju/model/unset.go for an example). We're best to be consistent. Also update the showMachineCommandDoc above